### PR TITLE
BaSP-TG-62: Shared button component

### PR DESCRIPTION
**Pull request by ProDIGGY-vinilos**
Shared button (primary and secondary version) with their styles and a rounded version ready to implement

### CODE REVIEW

File: src/Components/Shared/Button/primaryBtn.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Button/primaryBtn.js:1]: I see that the only difference between the two buttons is in the style. In this case it's better to have a single component called 'Button' and make a conditional to use one class or the other

File: src/Components/Shared/Button/primaryBtn.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { useHistory } from 'react-router-dom';
+import styles from './primaryBtn.module.css';
+
+const PrimaryBtn = (props) => {
+  let history = useHistory();
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Button/primaryBtn.js:6]: history should be a const

File: src/Components/Shared/Button/primaryBtn.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  const onAction = () => {
+    if (props.href != undefined) {
+      history.push(`${props.href}`);
+    } else {
+      props.onClick();
+    }
+  };
+  return (
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Button/primaryBtn.js:15]: Leave a blank line before the return

File: src/Components/Shared/Button/primaryBtn.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styles from './primaryBtn.module.css';
+
+const PrimaryBtn = (props) => {
+  let history = useHistory();
+
+  const onAction = () => {
+    if (props.href != undefined) {
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Button/primaryBtn.js:9]: Always use strict comparison !== or ===

File: src/Components/Shared/Button/primaryBtn.module.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    color: white;
+    border-radius: 100%;
+    text-decoration: none;
+    font-size: 25px;
+    transition-duration: 0.4s;
+}
+
+.rounded:hover {
+    cursor: pointer;
+    background-color: white;
```
Comment (Ignacio-Sormani) [inline src/Components/Shared/Button/primaryBtn.module.css:35]: Use hexa colors. For example: #FFFFFF or #000000

File: src/Components/SuperAdmins/index.js [hunk 6]
```diff
@@ -45,9 +45,9 @@ function SuperAdmins() {
           })}
         </tbody>
       </table>
-      <Link to="/super-admin-form">
-        <button>+</button>
-      </Link>
+      <div className={styles.addBtn}>
```
Comment (Ignacio-Sormani) [inline src/Components/SuperAdmins/index.js:48]: Is there a way to just use the Button without wrapping it in a div?

Comment (Ignacio-Sormani) [reply src/Components/SuperAdmins/index.js:48]: Maybe adding the className to the button

**Review by Ignacio-Sormani — CHANGES_REQUESTED** [review]

File: src/Components/Shared/Button/primaryBtn.js (hunk 2 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/Button/primaryBtn.js:6]: Done

File: src/Components/Shared/Button/primaryBtn.js (hunk 1 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/Button/primaryBtn.js:1]: Done

File: src/Components/Shared/Button/primaryBtn.js (hunk 4 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/Button/primaryBtn.js:9]: Done

File: src/Components/SuperAdmins/index.js (hunk 6 above)
Comment (ProDIGGY-vinilos) [reply src/Components/SuperAdmins/index.js:48]: i been trying but i cant make the button stay at the right, adding the className  to the Button doesn't take effect, the only thing i think is to adding the styles direct into the rounded style

File: src/Components/Shared/Button/primaryBtn.js (hunk 3 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/Button/primaryBtn.js]: Done

File: src/Components/Shared/Button/primaryBtn.module.css (hunk 5 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Shared/Button/primaryBtn.module.css]: Done

**Review by fede230298 — APPROVED** [review]

**Review by Ignacio-Sormani — APPROVED** [review]

**Pull request merged**